.github/workflows/caller_reach_out_to_zendesk_on_release_merge.yml

```diff
@@ -0,0 +1,20 @@
+name: Comment on related Zendesk tickets upon release
+#Description: Whenever the release branch is merged into main or master, call
+#the reusable_reach_out_to_zendesk_on_release_merge workflow in internal-actions-private.
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - main
+      - master
+jobs:
+  if_merged:
+    if: |
+      github.event.pull_request.merged == true &&
+      github.head_ref == 'release' &&
+      contains(github.event.pull_request.title, 'chore') &&
+      contains(github.event.pull_request.title, 'release')
```

Comment:
Do we know why this is contains and not starts with? This seems like it would run prematurely if the pull request title includes it beyond the semantic commit.